src/FsLibLog/FsLibLog.fs

```diff
@@ -46,8 +46,54 @@ module Types =
         abstract member Log :  Logger
         abstract member MappedContext :  MappedContext
 
+#if FABLE_COMPILER
+    // Fable doesn't support System.Collections.Generic.Stack, so this implementation (from FCS)
+    // is used instead.
+    type Stack<'a>(n)  =
+        let mutable contents = Array.zeroCreate<'a>(n)
+        let mutable count = 0
+
+        member buf.Ensure newSize =
+            let oldSize = contents.Length
+            if newSize > oldSize then
+                let old = contents
+                contents <- Array.zeroCreate (max newSize (oldSize * 2))
+                Array.blit old 0 contents 0 count
+
+        member buf.Count = count
+        member buf.Pop() =
+            let item = contents.[count - 1]
+            count <- count - 1
+            item
+
+        member buf.Peep() = contents.[count - 1]
+        member buf.Top(n) = [ for x in contents.[max 0 (count-n)..count - 1] -> x ] |> List.rev
+        member buf.Push(x) =
+            buf.Ensure(count + 1)
+            contents.[count] <- x
+            count <- count + 1
+
+        member buf.IsEmpty = (count = 0)
+#endif
+
     [<AutoOpen>]
     module Inner =
+#if FABLE_COMPILER
```

Comment:
Since the API is the same, the only thing that needs ifdef here is the ` open System.Collections.Generic` I think